.circleci/create_circleci_config.py

```diff
@@ -309,7 +309,7 @@ def job_name(self):
     docker_image=[{"image":"huggingface/transformers-torch-tf-light"}],
     install_steps=[
         "uv venv",
-        "uv pip install .[torch,tf,testing,sentencepiece,onnxruntime,vision,rjieba]",
```

Comment:
we don't really need to specify `torch,tf` as they are already installed during docker image built.

If we keep this step here, somehow torch will downgrade to 2.6.0 while torchvision/torchaudio keep the versions corresponding to torch 2.7.1 and we get mismatch issue, the whole job is failing.

I didn't dive into the details as we don't want to spend too much time on this onnx job.